platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/EnhancedKeyStoreLoader.java

```diff
@@ -349,6 +393,28 @@ public EnhancedKeyStoreLoader generate()
                         SecureRandom.getInstanceStrong());
                 agrCertificates.put(node, agrCert);
             }
+
+            if (!tssPrivateKeys.containsKey(node)) {
+                // Create a new public/private key pair for the TSS encryption key
+                BlsKeyPair tssKeyPair = CryptoStatic.generateBlsKeyPair(null);
```

Comment:
Updated to pass in SecureRandom.getInstanceStrong(). Originally had designed the method for callers to pass in null and it would use SecureRandom.getInstanceStrong as the cryptography library has methods BlsKeyPair.generate with different signatures that use a provided SecureRandom instance or SecureRandom.getInstanceStrong().